docs/reference_guide.md

```diff
@@ -324,6 +326,28 @@ Examples in situ:
 [search /examples](https://github.com/iovisor/bcc/search?q=bpf_log2l+path%3Aexamples&type=Code),
 [search /tools](https://github.com/iovisor/bcc/search?q=bpf_log2l+path%3Atools&type=Code)
 
+## Debugging
+
+### 1. bpf_override_return()
+
+Syntax: ```int bpf_override_return(struct pt_regs *, unsigned long rc)```
+
+Return: 0 on success
+
+When used in a program attached to a function entry kprobe, causes the
+execution of the function to be skipped, immediately returning `rc` instead.
+This is used for targeted error injection. 
+
+Note: bpf_override_return will only work when the kprobed function is
+whitelisted to allow error injections.
```

Comment:
Sorry, for the non-initiated, it would be very useful to explain how to discover whether a function is whitelisted for error injection, and what kind of error to expect if it is not whitelisted. Thanks!